caps/eth.md

```diff
@@ -108,22 +108,173 @@ relayed by the peer.
 Transactions must be validated before re-propagating them. Relaying an invalid transaction
 results in peer disconnection.
 
+### Transaction Encoding and Validity
+
+Transaction objects exchanged by peers have one of two encodings, which are identical to
+the encodings used by Ethereum consensus. In definitions across this specification, we
+refer to transaction objects of either encoding using the identifier `txₙ`.
+
+    tx = {legacy-tx, typed-tx}
+
+Untyped, legacy transactions are given as an RLP list.
+
+    legacy-tx = [
+        nonce: P,
+        gas-price: P,
+        gas-limit: P,
+        recipient: {B_0, B_20},
+        value: P,
+        data: B,
+        V: P,
+        R: P,
+        S: P,
+    ]
+
+[EIP-2718] typed transactions are encoded as RLP byte arrays where the first byte is the
+transaction type (`tx-type`) and the remaining bytes are opaque type-specific data.
+
+    typed-tx = rlp(tx-type || tx-data)
+
+Transactions must be validated when they are received. Validity depends on the Ethereum
+chain state. While the encoding of typed transactions is opaque, it is assumed that the
+encoding of `tx-data` provides values for `nonce`, `gas-price`, `gas-limit`, and that the
+sender account of the transaction can be determined from the signature.
+
+Transactions must be validated according to these rules:
+
+- If the transaction is typed, the `tx-type` must be one of the types considered
+  acceptable at the current block height.
+- The signature (`V`, `R`, `S` values) must be valid according to the signature schemes
+  supported by the chain. At this time, the two schemes in active use are the basic
+  'Homestead' scheme where `V`, `R`, `S` are simply the components of a secp256k1
+  signature; and the [EIP-155] scheme, where the signature values include a 'chain ID' to
+  prevent replay of the transaction on a different chain.
+- The `gas-limit` must cover the 'intrinsic gas' of the transaction, i.e. the base fee and
+  fee for the data size.
+- The sender account of the transaction, which is derived from the signature, must have
+  sufficient ether balance to cover the cost (`gas-limit * gas-price + value`) of the
+  transaction.
+- The `nonce` of the transaction must be equal or greater than the current nonce of the
+  sender account.
+- When considering the transaction for inclusion in the local pool, is up to
+  implementations to determine how many 'future' transactions with nonce greater than the
+  current account nonce are valid, and to which degree 'nonce gaps' are acceptable.
```

Comment:
I'm sensing a theme.  😀  Almost everything in this list should be left out of the networking spec because it is highly coupled with the individual transaction types.  I think the networking specification should be delegate responsibility for transaction validation down to other specifications with a simple statement like "The transaction must be validated according to the validation rules for the type of transaction received."

The `gas-limit * gas-price + value` is likely to break with account abstraction transaction types, and the nonce concept may change with future types as well.